kernel/univ.ml

```diff
@@ -345,8 +345,8 @@ struct
           (Level.is_prop u && not (Level.is_sprop v))
         else false
 
-    let successor (u,n) =
-      if Level.is_small u then type1
+    let successor (u,n as e) =
+      if is_small e then type1
```

Comment:
@gares maybe you will like this diff better than the previous one (this `is_small` is `Expr.is_small` from the current module)